src/Aspire.Hosting.AppHost/buildMultiTargeting/Aspire.Hosting.AppHost.targets

```diff
@@ -1,3 +1,3 @@
 <Project>
-  <Import Project="..\build\Aspire.Hosting.AppHost.targets" />
+  <Import Project="..\build\Aspire.Hosting.AppHost.in.targets" />
```

Comment:
We want to import the final version of the file. not the template.

Reply:
doh I thought I explicitly didn't update that one :\

Reply:
Fixed.